apps/wallet/src/ui/app/shared/button-connected-to/index.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
+import { type ComponentProps, forwardRef, type ReactNode } from 'react';
+
+const styles = cva(
+    [
+        'cursor-pointer outline-0 flex flex-row items-center py-1 px-2 gap-1 rounded-2xl',
+        'transition text-body-small font-medium border border-solid',
+        'border-transparent bg-transparent',
+        'hover:text-hero hover:bg-sui-light hover:border-sui',
+        'focus:text-hero focus:bg-sui-light focus:border-sui',
+        'active:text-steel active:bg-gray-45 active:border-transparent',
+        'disabled:text-gray-60 disabled:bg-transparent disabled:border-transparent',
+    ],
+    {
+        variants: {
+            bgOnHover: {
```

Comment:
Can we just bundle these into a cohesive `variant`? I actually imagine this entire component could just itself be a variant of the `Button` component.

Reply:
What do you mean with `a cohesive variant`? I wanted to keep the component a bit less complicated for now and create a Button component (and do any migrations) in a separate task (in interest of time). Hope it's not blocking and can address this later.